prov/rxd/src/rxd.h

```diff
@@ -397,29 +415,41 @@ static inline int rxd_match_tag(uint64_t tag, uint64_t ignore, uint64_t match_ta
 }
 
 int rxd_info_to_core(uint32_t version, const struct fi_info *rxd_info,
-		     struct fi_info *core_info);
+			struct fi_info *core_info);
 int rxd_info_to_rxd(uint32_t version, const struct fi_info *core_info,
-		    struct fi_info *info);
+			struct fi_info *info);
 
 int rxd_fabric(struct fi_fabric_attr *attr,
-	       struct fid_fabric **fabric, void *context);
+			struct fid_fabric **fabric, void *context);
 int rxd_domain_open(struct fid_fabric *fabric, struct fi_info *info,
-		    struct fid_domain **dom, void *context);
+			struct fid_domain **dom, void *context);
 int rxd_av_create(struct fid_domain *domain_fid, struct fi_av_attr *attr,
-		  struct fid_av **av, void *context);
+			struct fid_av **av, void *context);
 int rxd_endpoint(struct fid_domain *domain, struct fi_info *info,
-		 struct fid_ep **ep, void *context);
+			struct fid_ep **ep, void *context);
 int rxd_cq_open(struct fid_domain *domain, struct fi_cq_attr *attr,
-		struct fid_cq **cq_fid, void *context);
+			struct fid_cq **cq_fid, void *context);
 int rxd_cntr_open(struct fid_domain *domain, struct fi_cntr_attr *attr,
-		  struct fid_cntr **cntr_fid, void *context);
+			struct fid_cntr **cntr_fid, void *context);
 int rxd_query_atomic(struct fid_domain *domain, enum fi_datatype datatype,
-		     enum fi_op op, struct fi_atomic_attr *attr, uint64_t flags);
+			enum fi_op op, struct fi_atomic_attr *attr, uint64_t flags);
 
 /* AV sub-functions */
 int rxd_av_insert_dg_addr(struct rxd_av *av, const void *addr,
-			  fi_addr_t *dg_fiaddr, uint64_t flags,
-			  void *context);
+		fi_addr_t *dg_fiaddr, uint64_t flags,
+		void *context, size_t addrlen);
+
+/* AV/Peer subfunctions */
+struct rxd_peer *rxd_get_peer_entry(struct rxd_ep *ep, const void *addr, 
+			size_t addrlen);
+void rxd_fi_update_peer_entry(struct rxd_ep *ep, const void *addr,
+			fi_addr_t fi_addr, fi_addr_t *dg_addr, size_t addrlen);
+int rxd_fiaddr_hash_insert(struct rxd_ep *ep, fi_addr_t *fi_addr,
+			fi_addr_t *rxd_addr);
+int rxd_epaddr_hash_insert(struct rxd_ep *ep, void *addr,
+			fi_addr_t *rxd_addr);
+int rxd_update_av_peers(struct util_av *util_av, void *dg_addr,
+			fi_addr_t fi_addr, void *arg);
```

Comment:
These two functions I think are only called once each in rxd_ep. Remove them here and make them static in rxd_ep